en/enable-tls-for-mysql-client.md

```diff
@@ -23,6 +23,14 @@ To enable TLS for the MySQL client, perform the following steps:
 
 2. [Deploy the cluster](#deploy-the-tidb-cluster), and set `.spec.tidb.tlsClient.enabled` to `true`.
 
+    * To skip TLS authentication for internal components as the MySQL client (such as TidbInitializer, Dashboard, Backup, and Restore), you can add the `tidb.tidb.pingcap.com/skip-tls-when-connect-tidb="true"` annotation to the cluster's corresponding `TidbCluster`.
```

Comment:
```suggestion
    * To skip TLS authentication for internal components that serve as the MySQL client (such as TidbInitializer, Dashboard, Backup, and Restore), you can add the `tidb.tidb.pingcap.com/skip-tls-when-connect-tidb="true"` annotation to the cluster's corresponding `TidbCluster`.
```

Reply:
Perhaps, the updates are inaccurate, but "internal components as the MySQL client" looks confusing. If necessary, please rephrase further.

Reply:
I think your suggestion looks better. Thanks.